app/views/works/index.html.erb

```diff
@@ -0,0 +1,94 @@
+<h2>List of Works </h2>
+
+<section class="album">
+  <h3>Albums</h3>
+  <% albums = Work.where(media: "album")%>
+  <% albums = albums.sort_by {|work| work.votes.length}.reverse%>
```

Comment:
It may make more sense to put this logic in the controller, and then the eventual value of `albums` as an instance variable `@albums`

Reply:
You defined the method `top_ten` in your `Work` model, but didn't use it here! :O